### PR TITLE
improve doc about SESSION_PROTECTION setting

### PR DESCRIPTION
**Pull request by yanxurui**

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -397,9 +397,10 @@ Or, to disable it::
 
     login_manager.session_protection = None
 
-By default, it is activated in ``"basic"`` mode. It can be disabled in the
+By default, it is activated in ``"basic"`` mode. It can also be configured in the
```
Comment (alanhamlett) [inline docs/index.rst]: This line is longer than 80 characters, please wrap lines longer than 80 chars.

**Comment by alanhamlett** [comment]
I'm not sure how this is better. Every config overwrites settings on `LoginManager` so stating that `SESSION_PROTECTION` overwrites `LoginManager.session_protection` sounds like it behaves different when it doesn't.

**Comment by yanxurui** [comment]
Hello,  according to the source code default configuration are provided in config.py except  `SESSION_PROTECTION` which falls back to login manager's `session_protection` attribute.
That's why I think it deserves stating additionally. Actually, I suggest only configuring by Flask app's configuration.

**Comment by yanxurui** [comment]
I am sorry to disturb you all. I will close this pull request if it doesn't help.

**Pull request closed without merging**